samples/client/echo_api/php-nextgen-streaming/src/ObjectSerializer.php

```diff
@@ -268,6 +268,11 @@ public static function toQueryValue(
 
         $value = $flattenArray($value, $paramName);
 
+        // https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#style-values
+        if ($openApiType === 'array' && $style === 'deepObject' && $explode) {
+            return $value;
+        }
```

Comment:
This change looks like to be not related to the PR.

Reply:
An easy fix would be just to remove it. Check there is no difference with master using `git diff martin...patch-1` before committing.